packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -221,11 +239,51 @@ export default function Sidebar({
               <MyFeedButton
                 {...defaultRenderSectionProps}
                 isButton={isNavButtons}
-                alerts={alerts}
+                title="My feed"
+                path="/"
+                alert={
+                  (alerts.filter || alerts.myFeed) &&
+                  !sidebarExpanded && (
+                    <AlertDot
+                      className="right-2.5 top-0"
+                      color={AlertColor.Success}
+                    />
+                  )
+                }
                 onNavTabClick={onNavTabClick}
                 icon={<ProfilePicture size="xsmall" user={user} />}
               />
             )}
+            <MyFeedButton
+              {...defaultRenderSectionProps}
+              isButton={false}
+              title="New feed"
+              path={`${webappUrl}feeds/new`}
+              icon={
+                <div className="rounded-6 bg-background-subtle">
+                  <PlusIcon />
+                </div>
+              }
+              rightIcon={() =>
+                showCustomFeedsBetaBadge ? (
+                  <div className="absolute -right-3 -top-2 h-[16px] w-[41px]">
+                    <img src={cloudinary.feed.betaTag} alt="Beta" />
+                  </div>
+                ) : undefined
+              }
+            />
+            {feeds?.edges?.map((feed) => {
+              return (
+                <MyFeedButton
+                  {...defaultRenderSectionProps}
+                  key={feed.node.id}
+                  isButton={false}
+                  title={feed.node.flags.name || `Feed ${feed.node.id}`}
+                  path={`${webappUrl}feeds/${feed.node.id}`}
+                  icon={<StarIcon secondary />}
```

Comment:
Just noting it will probably change to be the outline by default and filled when active.
https://dailydotdev.slack.com/archives/C06LKPZAZ1T/p1716371173254099